src/Model/Subsite.php

```diff
@@ -666,16 +667,10 @@ public function canEdit($member = false)
     public function getCMSFields()
     {
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            if ($this->exists()) {
-                // Add a GridField for domains to a new tab if the subsite has already been created
-                $fields->addFieldsToTab('Root.Domains', [
-                    GridField::create(
-                        'Domains',
-                        '',
-                        $this->Domains(),
-                        GridFieldConfig_RecordEditor::create(10)
-                    )
-                ]);
+            $domainsGridField = $fields->dataFieldByName('Domains');
+            if ($domainsGridField instanceof GridField) {
+                $domainsGridField->setTitle('');
+                $domainsGridField->getConfig()->removeComponentsByType(GridFieldAddExistingAutocompleter::class);
```

Comment:
Why are we removing the GridFieldAddExistingAutocompleter?

Reply:
To match the behaviour that existing prior to this PR.